examples/api/amp-inputmask.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const express = require('express');
+const multer = require('multer');
+const upload = multer();
+
+// eslint-disable-next-line new-cap
+const examples = express.Router();
+
+examples.all('/default', upload.none(), (request, response) => {
```

Comment:
Why not use `examples.post` in the first place? Also valid for the other routes.

Reply:
This makes total sense, but the original amp-inputmask.go has a specific response to all request that are not via POST. If that is not necessary, we could switch over to your suggestion.

Reply:
Please do as Matthias suggested, the original implementation was due to Go's more rudimentary request handling APIs.